aten/src/ATen/native/SummaryOps.cpp

```diff
@@ -20,7 +20,7 @@ Tensor _bincount_cpu_template(
     AT_ERROR("minlength should be >= 0");
   }
   if (self.dim() == 1 && self.numel() == 0) {
-    return native::zeros({minlength}, kLong);
+    return at::zeros({minlength}, kLong);
```

Comment:
A modest improvement that would eliminate redispatching here would be to call `at::compositeexplicitautograd::zeros` but... ugh this name is so awful lol. It may be better to convert this namespace into entries for `at::cpu` and `at::cuda` for symmetry (and to help people ensure that those shortcuts always exist.)

Reply:
I think the idea is that here you're already in the CPU kernel (for `at::bin count`), so any SymInt/int64_t concerns are irrelevant now (you're in this CPU kernel, so you must not be tracing). And we could speed up this code a bit by skipping the dispatcher overhead when we call into the `zero()` kernel; `at::cpu::zeros` doesn't (conveniently) exist today. Agreed that it would be helpful to add it though.